Add a KaTeX and MathJax based LaTeX pane

philippjfr opened the pull request:
I think this should probably replace the LaTeX pane because it is much more capable. That said I still need to assess how expensive it would be to load KaTeX all the time, if it's too large I will likely have to let the user request it explicitly using ``pn.extension('katex')`` (not required on bokeh server or when saving).

![Screen Shot 2019-03-16 at 2 45 36 AM](https://user-images.githubusercontent.com/1550771/54469845-ef2e5100-4795-11e9-8002-a0964ab6017b.png)

philippjfr commented:
katex.min.js + katex.min.css are ~137 KB combined. Is it worth it to always bundle that or should I require the user to run ``pn.extension('katex')`` before using the component in the notebook? A helpful warning would tell the user they forgot to do so. I'm on the edge, I don't quite think using LaTeX is common enough to make everyone pay a 137 KB penalty. If I make it optional I could also allow toggling between MathJax and KaTeX and use whichever one is loaded.

jbednar commented:
I'd say it's worth saving the 137KB at a cost of having to load the extension explicitly

philippjfr commented:
Okay, I've now implemented MathJax as well and refactored in such a way that the LaTeX pane will use either the MathJax or the KaTeX model depending on which was loaded. If an explicit renderer is defined on the pane it will use that and if neither is loaded it will warn.
